cmd/api-errors.go

```diff
@@ -399,6 +400,11 @@ var errorCodes = errorCodeMap{
 		Description:    "Your proposed upload exceeds the maximum allowed object size.",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrPolicyTooLarge: {
+		Code:           "PolicyTooLare",
+		Description:    "Policy object exceeds the maximum allowed object size.",
```

Comment:
We should say policy size not object size..

Reply:
Changed into `document size`